app/app.go

```diff
@@ -399,7 +399,7 @@ func New(
 	// Create Transfer Keepers
 	app.TransferKeeper = ibctransferkeeper.NewKeeper(
 		appCodec, keys[ibctransfertypes.StoreKey], app.GetSubspace(ibctransfertypes.ModuleName),
-		app.IBCKeeper.ChannelKeeper, &app.IBCKeeper.PortKeeper,
+		app.IBCKeeper.ChannelKeeper, app.IBCKeeper.ChannelKeeper, &app.IBCKeeper.PortKeeper,
```

Comment:
why 2 ChannelKeeper?

Reply:
Its because we moved out the ICS4 (middleware) interface from the channel
https://github.com/cosmos/ibc-go/blob/v3.0.0/modules/apps/transfer/keeper/keeper.go#L35

We don't have any ibc middleware yet so we are just using the channel base implementation